src/pages/ContactPage.tsx

```diff
@@ -1,5 +1,157 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  HStack,
+  Input,
+  Text,
+  Textarea,
+} from "@chakra-ui/react";
+import { ErrorMessage } from "@hookform/error-message";
+import { FieldValues, useForm } from "react-hook-form";
+import { TRANSITION_SLOW } from "../consts";
+
 const ContactPage = () => {
-  return <div>Contact Page</div>;
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm({ criteriaMode: "all" });
+  const onSubmit = (event: FieldValues) => {
+    console.log(event);
+  };
+
+  return (
+    <Box w={"100%"} my={"auto"} p={16} id="contact-form-container" pt={0}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <HStack>
+          <FormControl p={2}>
+            <FormLabel>First Name</FormLabel>
+            <Input
+              {...register("firstName", {
+                required: "Please enter your first name",
+                pattern: {
+                  value: /^[A-Za-z]+$/,
+                  message: "Please enter a valid first name",
+                },
+              })}
+              variant={"filled"}
+            />
+            {!errors.firstName ? <Box h={"24px"}></Box> : null}
+            <ErrorMessage
+              errors={errors}
+              name="firstName"
+              render={({ messages }) => {
+                return messages
```

Comment:
maybe we can extract this out in a method as it is common across all input elements

Reply:
Agreed. There are a few refactoring opportunities in the entire page component. We can discuss these and pick it as a separate task. For the time being we can move ahead with the merge.